packages/trip-form/src/TripOptions/index.tsx

```diff
@@ -56,19 +56,31 @@ interface ComponentProps {
   featuredItemOverlayShown?: (overlayShown: boolean) => void;
 
   /**
-   * Icon prop used for overwriting the question mark icon throughout the component
+   * An optional prop to override svg fill color of CompanyIcons
+   * Note: this will only work if the image field of the option is an svg
    */
-  QuestionIcon: ReactElement;
+  tripOptionIconFillOverride?: string;
 
   /**
-   * Icon prop used for overwriting mode icons throughout the component
+   * Object of icon props used to overwrite the checkmark and plus icons
    */
-  SimpleModeIcon?: FunctionComponent<{ mode: string }>;
+  checkboxIcons?: { checked: FunctionComponent; unchecked: FunctionComponent };
```

Comment:
Sort props.

Reply:
This was a tough one but sorted in a10727e